Python/Product/PythonTools/PythonTools/Project/ConfigureTestFrameworkInfoBar.cs

```diff
@@ -281,6 +278,15 @@ internal sealed class TestFrameworkProjectInfoBar : TestFrameworkInfoBar {
             Project.SetProjectProperty(propertyName, propertyValue);
             return Task.CompletedTask;
         }
+
+        protected override bool PythonTestFileFound(Predicate<string> fileFilter) {
+            return Project.AllDescendants
+                .Where(x => (x is PythonFileNode || x is PythonNonCodeFileNode))
```

Comment:
did you test that this correctly returns false for files that are excluded from project?

Reply:
Yes